libcxx/test/libcxx/odr_signature.exceptions.sh.cpp

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// TODO: Investigate
+// XFAIL: target={{.+}}-windows-{{.+}}
```

Comment:
@mstorsjo Do you know why this might fail on Windows? ABI tags are supported an honored on Windows too, right?

Reply:
It seems like these ABI tags aren't supported when using the MSVC C++ name mangling - but as you observed, it does work with MinGW which uses the regular Itanium name mangling. I don't know the details any further than that for the MSVC C++ ABI though - maybe @rnk might know?